src/common/transformations/include/transformations/rt_info/disable_fp16_compression.hpp

```diff
@@ -39,7 +39,7 @@ class TRANSFORMATIONS_API DisableFP16Compression : public RuntimeAttribute {
     }
 
     bool is_copyable() const override {
-        return false;
```

Comment:
looks not safe
As far as I remember, this attribute affects several transformations
Did we run any validation for this change?

Reply:
it was needed in my very fist draft solution but now it's not necessary. 

Ok, i got it, it's dangerous to change it now: we have a separate ticket for making in copyable CVS-130863